ui/src/data_explorer/containers/DataExplorer.tsx

```diff
@@ -158,9 +160,11 @@ export class DataExplorer extends PureComponent<Props, State> {
     } = this.props
     const {query, script} = this.queryString
 
-    GlobalAutoRefresher.poll(autoRefresh)
+    await this.fetchFluxServices()
 
-    if (script || _.isEmpty(query)) {
+    if (script) {
```

Comment:
This logic is getting a little difficult to decipher. Not req'd but I think it might benefit from naming the booleans? 
like `isFlux = !!script` to make it more legible.

Reply:
👍